pyiceberg/io/fsspec.py

```diff
@@ -150,6 +151,9 @@ def _s3(properties: Properties) -> AbstractFileSystem:
     if connect_timeout := properties.get(S3_CONNECT_TIMEOUT):
         config_kwargs["connect_timeout"] = float(connect_timeout)
 
+    if request_timeout := properties.get(S3_REQUEST_TIMEOUT):
+        config_kwargs["request_timeout"] = float(request_timeout)
```

Comment:
I think this should be `read_timeout`, looking at: https://github.com/fsspec/s3fs/blob/51e3c80ef380a82081a171de652e2b699753be2b/s3fs/core.py#L473-L479

Reply:
@Fokko You're entirely correct - thanks for catching that! I'll make the correction and push the updated code.